ports/mimxrt/boards/MAKERDIARY_RT1011_NANO_KIT/pins.csv

```diff
@@ -0,0 +1,43 @@
+A0,GPIO_AD_00
+A1,GPIO_AD_01
+A2,GPIO_AD_02
+A3,GPIO_AD_03
+A4,GPIO_AD_04
+A5,GPIO_AD_05
+A6,GPIO_AD_06
+A7,GPIO_AD_07
+A8,GPIO_AD_08
+A9,GPIO_AD_09
+A10,GPIO_AD_10
+A11,GPIO_AD_11
+A14,GPIO_AD_14
+
+D0,GPIO_00
+D1,GPIO_01
+D2,GPIO_02
+D3,GPIO_03
+D4,GPIO_04
+D5,GPIO_05
+D6,GPIO_06
+D7,GPIO_07
+D8,GPIO_08
+D9,GPIO_09
+D10,GPIO_10
+D11,GPIO_11
+D12,GPIO_12
+D13,GPIO_13
+
+SD0,GPIO_SD_00
+SD1,GPIO_SD_01
+SD2,GPIO_SD_02
+SD5,GPIO_SD_05
+# Test pads at the bottom side
+SD3,GPIO_SD_03
+SD4,GPIO_SD_04
+SD12,GPIO_SD_12
+SD13,GPIO_SD_13
+
+
+LED,GPIO_SD_04
+CLK,GPIO_AD_12
+DIO,GPIO_AD_13
```

Comment:
Is it worth calling these last two `SWDCLK` and `SWDIO`?

I know they are CLK/DIO on the silk screen (which is what we usually want these names to match).  But the pinout diagram calls them SWDCLK/SWDIO, and calling them that in pins.csv might be a bit easier for users to discover (I initially thought they were an RGB CLK and data line).

I leave the decision up to you.

Reply:
I do not mind either. The preference is for the silkscreen version, because that's what users definitely can see. But I can as well add both versions CLK/DIO and SWCLK/SWDIO.

Reply:
The label of the Pin is used in Python code not as Debug pin, but for instance as a machine.Pin object. For that use, the silkscreen text is consistent with the other pins. 

And yes, when hooking up the SWD interface I had a short moment fo doubt whether the CLK/DIO pins are SWCLK/SWDIO.  But that is a different story.

Reply:
Let's leave it as the silkscreen text.

Reply:
OK